### PR TITLE
Migration to `core-java` 0.8.0

### PR DESCRIPTION
**Pull request by dmytro-grankin**
Summary:

- Fix `event store` example.
- Migrate to `core-java` 0.8.0.
- Add `Hello, World!` sample.

### CODE REVIEW

**Comment by armiol** [comment]
@dmytro-grankin please fix the issues reported by Codacy. We need this check to pass as well.

File: hello-world/src/main/proto/hello_world/command.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+package hello_world;
+
+option (type_url_prefix) = "type.spine3.org";
+option java_package="org.spine3.helloworld.commands";
+option java_multiple_files = true;
+
+import "spine/annotations.proto";
+import "spine/users/user_id.proto";
+
+// Command represents protobuf message, that contains target entity ID and username for introduction.
```
Comment (armiol) [inline hello-world/src/main/proto/hello_world/command.proto]: `protobuf` is obvious, as it is a `.proto` file.

Comment (armiol) [inline hello-world/src/main/proto/hello_world/command.proto]: Please add the required articles. Currently the grammar is poor.

File: hello-world/src/main/proto/hello_world/event.proto [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+package hello_world;
+
+option (type_url_prefix) = "type.spine3.org";
+option java_package="org.spine3.helloworld.events";
+option java_multiple_files = true;
+
+import "spine/annotations.proto";
+import "spine/users/user_id.proto";
+
+// Event represents protobuf message, that contains only introduced username.
```
Comment (armiol) [inline hello-world/src/main/proto/hello_world/event.proto]: Same as above.

Comment (armiol) [inline hello-world/src/main/proto/hello_world/event.proto]: `.. the name of the introduced user.`

File: hello-world/src/main/proto/hello_world/command.proto (hunk 1 above)
Comment (dmytro-grankin) [inline hello-world/src/main/proto/hello_world/command.proto]: Fixed.

Comment (dmytro-grankin) [inline hello-world/src/main/proto/hello_world/command.proto]: Done.

File: hello-world/src/main/proto/hello_world/event.proto (hunk 2 above)
Comment (dmytro-grankin) [inline hello-world/src/main/proto/hello_world/event.proto]: Done.

Comment (dmytro-grankin) [inline hello-world/src/main/proto/hello_world/event.proto]: Done.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: hello-world/src/main/proto/hello_world/event.proto [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+option (type_url_prefix) = "type.spine3.org";
+option java_package="org.spine3.helloworld.events";
+option java_multiple_files = true;
+
+import "spine/annotations.proto";
+import "spine/users/user_id.proto";
+
+// The message to create the event. Contains only the name of the introduced user.
+// The username is all required information just for print `Hello, ...!`.
```
Comment (armiol) [inline hello-world/src/main/proto/hello_world/event.proto]: The grammar is still far from perfect. Let's remove this line at all.

Please refer to the [events declared in Spine `todo-list`](https://github.com/SpineEventEngine/todo-list/blob/master/model/src/main/proto/todolist/c/events.proto) for an example of a good description.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: hello-world/src/main/proto/hello_world/command.proto [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+package hello_world;
+
+option (type_url_prefix) = "type.spine3.org";
+option java_package="org.spine3.helloworld.commands";
+option java_multiple_files = true;
+
+import "spine/annotations.proto";
+import "spine/users/user_id.proto";
+
+// Introduce an user with a name
```
Comment (armiol) [inline hello-world/src/main/proto/hello_world/command.proto]: Please end all the sentences with a dot.

Comment (dmytro-grankin) [inline hello-world/src/main/proto/hello_world/command.proto]: Done.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**